langkit/dsl_unparse.py

```diff
@@ -973,6 +973,8 @@ def bases(typ):
             return type.raw_name.camel
```

Comment:
Given that the previous implementation, present in this PR, was wrong, what about squashing this commit?